### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by polRk**
Add usage section

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,22 @@
 Rust SDK for YDB.
 Supported rust: 1.60.0 and newer.
 
+## Usage
```
Comment (vgvoleg) [inline README.md:9]: Installation

File: README.md [hunk 2]
```diff
@@ -6,12 +6,22 @@
...
+Add this to your Cargo.toml:
+
+```toml
+[dependencies]
+ydb = "0.9.4"
+```
+
```
Comment (vgvoleg) [inline README.md:16]: let's add `cargo add ydb` as well

File: README.md [hunk 3]
```diff
@@ -4,14 +4,20 @@
...
-1. Set YDB_CONNECTION_STRING env
-2. run cargo test -- --include-ignored
+### Prerequisites
+Rust 1.60.0 or newer
+
+### Installation
+Add the YDB dependency to your project using `cargo add ydb` or add this your Cargo.toml:
+```toml
+[dependencies]
+ydb = "0.9.4"
```
Comment (rekby) [inline README.md:15]: It needs write common version, for example "0.x.y" or update script for update versions while release:
https://github.com/ydb-platform/ydb-rs-sdk/blob/6bbdf8a3736126879be09720c1e746dc487df65a/.github/scripts/version-up.sh

**Review by rekby — COMMENTED** [review]
@polRk 
Hello, thanks for the PR and improve documentation.

The SDK have script for auto update package version while release.

It needs to update the README too or write common version instead of current version.

**Review by rekby — APPROVED** [review]
thanks for your pr :)

**Pull request merged**